tide_publication.install

```diff
@@ -276,3 +276,23 @@ function tide_publication_update_8005() {
   $config->set('cardinality', -1);
   $config->save();
 }
+
+/**
+ * Add re-order children permission for roles.
+ */
+function tide_core_update_8006() {
```

Comment:
@MdNadimHossain Does this belong in dpc-sdp/tide_core?

Role names shouldn't be hardcoded other than in the module that defines them.

Reply:
I gave the hook name wrong :/ and I have added this permission in tide_publication because this permission is related to tide_publication and enity_hirarchy module also a dependency of this module.

Reply:
As we discussed in Teams because this should also be added during install keeping this in tide_publication makes sense. Because this module has a dependency on tide_landing_page which in turn has a dependency on tide_core we can rely on those roles existing. It's not ideal but it's the most practical solution.